cookbook/agents/agentic_search/lightrag/agentic_rag_with_lightrag.py

```diff
@@ -17,22 +17,22 @@
 )
 
 
-knowledge.add_content(
+asyncio.run(knowledge.add_content(
     name="Recipes",
     path="cookbook/knowledge/data/filters/cv_4.pdf",
     metadata={"doc_type": "recipe_book"},
```

Comment:
We added `add_contents_sync` to make this simpler, would be cool to show it off in these cookbooks